objects/cash_register.rb

```diff
@@ -0,0 +1,24 @@
+#!/usr/bin/env ruby
+
+# This is the CashRegister class
+class CashRegister
+  attr_reader :purchase, :total, :pay
```

Comment:
You still don't need to purchase and pay methods here. If you're not 100% sure why, ask me in class